### DISCUSSION TITLE
More stock?

### DISCUSSION THREAD

**Discussion by woody55uk** [nplan/HomeButtons#78] [General]
Hi. Any plans to make more Home Buttons - they seem to have been out of stock for a while.

Mike

**Comment by nplan** [comment]
Hi, the 6 button version is in stock on [Tindie](https://www.tindie.com/products/28406/). The Mini is not available at the moment.

**Reply by woody55uk** [reply]
Thanks. It was the mini I wanted. Are you planning on making any more? If so I'll sign up for the waitlist on Tindie.

Mike

**Reply by nplan** [reply]
Please sign up for the waitlist. Right now there are no plans to make more because the 6 button version is more popular. But if there's enough interest I will make another batch.

**Reply by woody55uk** [reply]
OK - done. Fingers crossed :)
